system/config/templates/features/quoting_tool_app.yml

```diff
@@ -64,4 +64,6 @@ registry:
           - key: :oe_maximum_length_months
             item: 2
           - key: :earliest_start_prior_to_effective_on_months
-            item: -2
\ No newline at end of file
+            item: -2
+      - key: :display_sic_codes
+        is_enabled: false
```

Comment:
can you remove `display` from key name. we also use sic codes while calculating rates. we should be using same flag for this as well.

Reply:
Done 👍